src/components/ResourceField.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {
+  Form,
+} from '../ant';
+import {Select} from 'antd'
+const {Option} = Select;
```

Comment:
This should be:
```
import {
  Form,
  Select,
} from '../ant';
const {Option} = Select;
```

Reply:
The reasoning behind this is that we are using `../ant` rather than `antd` to make some import statements easier in the long run.